docs/src/hugo/content/getting-started/first-use.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: "single"
+title: First Use
+preamble: >
+  Launching your first system using Nelson is remarkably easy! This guide walks you through how to get-going with Nelson and covers the high-level components needed to get an end to end solution.
+contents:
+- Assumptions
+- Manifest
+- Enabling Nelson
+- Triggering a Deployment
+- Working with Stacks
+- Further Reading
+menu:
+  main:
+    parent: gs
+    identifier: gs-first-use
+    url: /getting-started/first-use.html
+    weight: 2
+---
+
+## Assumptions
+
+Whilst Nelson is typically non-perscriptive, for the unfamiliar user and for the remainder of this article, the following assumptions will be made to avoid confussion:
+
+* You can launch any container you like using Nelson; there is no requirement from the system perspective that the container behaves in a particular way other than having a [typical Nelson entrypoint](https://docs.docker.com/engine/reference/builder/#entrypoint).
+
+* Nelson is setup and running in your environment, and configured to talk to a scheudler of your choosing.
+
+## Manifest
+
+One of the core tenets of the Nelson philosophy is that all changes are checked into source code - nothing should be actioned out of band, resutling in an untrackable system state. Nelson requires users define a manifest file and check it into the root of the source repository. This file is called `.nelson.yml` (note the preceding `.` since it is a UNIX dotfile).
+
+This file contains the application `unit` definitions, along with any additional configuration for monitoring, alerting and scaling those units as needed. Consider this example that launches a Hello World service:
```

Comment:
In our internal guide we have a table of some basic Nelson terminology, namely what a "unit" is versus a "stack", might be worth adding before we talk about "units"